css/style.css

```diff
@@ -16,4 +16,396 @@ body {
+}
+.social-links{
+    display:flex;
+    flex-direction: row;
+    flex-wrap:wrap;
+    padding:0;
+    margin:0 0.9em;
+    justify-content:center;
+}
+.social-links a{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 2.80em;
+    margin-right: 1em;
+    margin-bottom: 1rem;
+    width: 2.80em;
+    border:1px solid #9c9c9c;
+    border-radius: 25%;
+    padding: 0.9em;
+    color:rgb(238, 255, 0);
+    background-color: rgb(223, 202, 202);
+   transition: all 0.5s ease-in-out;
+}
+.social-links img{
+    height: 2em;
+    margin: 0 0.5em;
+} 
+.copyright{
+    font-size: 0.8em;
+    color: #424242;
+    margin-top: 0.5rem;
+}
+
+/**************** media query styles ***********************/
+
+@media (min-width:641px){
+    #intro{
+        height: 350px;
+    }
+    /*service section*/
+
+    .service-card img{
+    width: 150px;        
+    margin: 10px 15%;
+    }         
+}
+    /********/
+@media (min-width:961px){
+    .logo_pic{
+        height: 8vh;          
+}   
+    .nav-link{
+        display:flex ;
+        flex-direction: row;
+        position: relative;
+        justify-content: flex-end;
+        background-color: var(--primary);
+        top:0;
+        text-align: center;
+        margin-right: 25px;
+        transition: none;
+}
+    .nav-item{
+        padding: 0.6em;
+        margin: 0.5em;
+}
+    .toggle-button{
+        display: none;
+}
+    /*intro section*/
+    
+    #intro{
+        height: 600px;
+}
+
+    .title{
+        font-size: 4em;
+        letter-spacing: 0.1em;
+}
+    .description{
+        font-size: 1.8em;
+}
+    #intro .btn{
+        font-weight: 500;
+        padding:0.5em;
+        font-size: 1.5em;
+}
+    /*service section*/
+
+    #service-container h2{
+        text-align: center;
+        margin-top: 0.7em;
+        font-weight:400 ;
+        font-size:2em ;  
+    }
+    .cards {
+        display: flex;
+        flex-flow: row wrap;
+        justify-content: space-evenly;
+    }
+    .service-card{
+        max-width: 40%;
+        padding: 2em;
+    }
+    .service-card img{
+        max-width: 360px;       
+        margin: 10px 5%;
+    }
+    .service-card h3{
+        font-size: 1.7em;
+        font-weight: 300;
+        margin:0.5em 0;
+    }
+    /*get karma*/
+    
+    #get_karma{
+        background-color: rgb(241, 229, 205);
+        display: grid;
+        margin-bottom: 2em;
+    }
+    #get_karma img{
+        grid-column: 1 / 2;
+        grid-row: 1 / 2;
+    }
+    #get_karma h4{
+        grid-column: 2 / 4;
+        grid-row: 1 / 2;
+        justify-self: center;
+        align-self: center;
+        display: flex;
+        flex-direction: column;
+        margin: 0.8em 1.5em;
+        text-align: center;
+        font-style: italic;
+        font-weight: 400;
+        font-size: 1.5em;
+        color: var(--secondary);
+          
+    }
+    .karma-btn{
+        grid-column: 2 / 4;
+        grid-row:  1;
+        margin-top: 10em;
```

Comment:
it seems like this margin is being used to correct an issue in the grid layout - it looks fine but it would be better to work on the layout so that it is not required. This would make it easier to make changes to the layout and dimensions without needing to tweak the margin-top value

Reply:
Hi Paul, Thank you for your comment. I deleted the margin. And added the grid template column in the get karma section container. I hope I have done the right this time. I do struggle with lay out issues. I will practise more and  more . Kind regards.